src/transformers/models/bigbird_pegasus/tokenization_bigbird_pegasus.py

```diff
@@ -0,0 +1,80 @@
+# coding=utf-8
```

Comment:
think we can delete this class completely and re-use `PegasusTokenizer` -> I'll help you with it though :-)